web/views/days_view.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Janitor.DaysView do
+  use Janitor.Web, :view
+
+  def render("days.json", %{days: days}) do
```

Comment:
Please drop `"days.json"` as an argument. I don't think this is a good use of pattern matching.